pkg/api/apis/operators/installplan_types.go

```diff
@@ -248,6 +251,29 @@ func (b *BundleLookup) SetCondition(cond BundleLookupCondition) BundleLookupCond
 	return cond
 }
 
+func OrderSteps(steps []*Step) []*Step {
```

Comment:
I think a nice future feature would be to allow users to indicate sort order - maybe this is something like a runlevel annotation on resources. 

(this looks good, just looking forward)